src/platform/drivers/mss_can/mss_can.h

```diff
@@ -776,8 +776,8 @@ typedef union buffer_status
 #else
 typedef struct buffer_status
 {
-    __I uint32_t RXMSGAV;
-    __I uint32_t TXREQ;
+    __IO uint32_t RXMSGAV;

```

Comment:
#define __I  const volatile
#define __IO volatile
The buffer status registers are read only and hence needs to remain __I.